Demo/src/main/java/com/braintreepayments/demo/VenmoFragment.java

```diff
@@ -102,7 +102,7 @@ public void launchVenmo(View v) {
         ArrayList<VenmoLineItem> lineItems = new ArrayList<>();
         lineItems.add(new VenmoLineItem(VenmoLineItem.KIND_CREDIT, "Some Item", 1, "2"));
         lineItems.add(new VenmoLineItem(VenmoLineItem.KIND_DEBIT, "Two Items", 2, "10"));
-        venmoRequest.setLineItems(lineItems);
+        venmoRequest.setVenmoLineItem(lineItems);
```

Comment:
The method name was changed from setLineItem to setVenmoLineItems to more accurately reflect its functionality, and found this name more relatable. The previous name was ambiguous because it did not specify the type of line items being set. In reality, the method was responsible for handling a collection of VenmoLineItem.